adr/ADR-32.md

```diff
@@ -209,6 +218,9 @@ The type for this is `io.nats.micro.v1.ping_response`.
     /**
     * The number of requests received by the endpoint
     */
+    /**
+    * Queue group to which this endpoint is assigned to
+    */
```

Comment:
Something not quite right here formatting wise and probably also forgot the struct value here

Reply:
fixed.